app/models/text_field_type.rb

```diff
@@ -25,14 +29,22 @@ def mapping_field_name
   end
 
   def text_present
-    errors.add(:text, "must be present") if @text.empty?
+    errors.add(:text, 'must be present') if @text.empty?
   end
 
   def text_length
     validator = LengthValidator.new(validations[:length].merge(attributes: [:text]))
     validator.validate_each(self, :text, text)
   end
 
+  def text_unique
+    errors.add(:text, 'Slug Must be unique') unless FieldItem.jsonb_contains(:data, text: text).empty?
```

Comment:
@toastercup in regards to `require 'jsonb_accessor'` for some reason the FieldType sub classes don't have access to jsonb methods unless it's in Cortex Gemfile or explicitly required. Not sure why but that was the case